golangci-lint error in cimg-go:1.19

## Meta:
Operating System: OSX Monterey 12.0.1, Intel chip

## Current behavior:
I can't run golangci-lint with cimg/go:1.19.0.

```
docker run -it cimg/go:1.19.0

circleci@61aaede53bc0:~/project$ golangci-lint version
panic: load embedded ruleguard rules: rules/rules.go:13: can't load fmt

goroutine 1 [running]:
github.com/go-critic/go-critic/checkers.init.22()
	github.com/go-critic/go-critic@v0.6.3/checkers/embedded_rules.go:47 +0x4b4
```

--- a/1.19/Dockerfile
+++ b/1.19/Dockerfile
@@ -21,8 +21,8 @@ RUN sudo apt-get update && sudo apt-get install -y --no-install-recommends \
 	mkdir -p /home/circleci/go/bin
 
 # Install related tools
-ENV GOTESTSUM_V=1.8.1
-ENV GOCI_LINT_V=1.47.3
+ENV GOTESTSUM_V=1.8.2
+ENV GOCI_LINT_V=1.48.0
 RUN curl -sSL "https://github.com/gotestyourself/gotestsum/releases/download/v${GOTESTSUM_V}/gotestsum_${GOTESTSUM_V}_linux_amd64.tar.gz" | \
 	sudo tar -xz -C /usr/local/bin gotestsum && \
 	curl -sSfL https://raw.githubusercontent.com/golangci/golangci-lint/master/install.sh | sudo sh -s -- -b /usr/local/bin v${GOCI_LINT_V}

--- a/Dockerfile.template
+++ b/Dockerfile.template
@@ -22,7 +22,7 @@ RUN sudo apt-get update && sudo apt-get install -y --no-install-recommends \
 
 # Install related tools
 ENV GOTESTSUM_V=1.8.2
-ENV GOCI_LINT_V=1.47.3
+ENV GOCI_LINT_V=1.48.0
 RUN curl -sSL "https://github.com/gotestyourself/gotestsum/releases/download/v${GOTESTSUM_V}/gotestsum_${GOTESTSUM_V}_linux_amd64.tar.gz" | \
 	sudo tar -xz -C /usr/local/bin gotestsum && \
 	curl -sSfL https://raw.githubusercontent.com/golangci/golangci-lint/master/install.sh | sudo sh -s -- -b /usr/local/bin v${GOCI_LINT_V}

